scripts/updateUsers.ts

```
@@ -212,16 +61,162 @@ async function main() {
+          const workerRepo = manager.getRepository(Worker);
+          const workerStatsRepo = manager.getRepository(WorkerStats);
+
+          for (const result of fetched) {
+            const address = result.address;
+            if (result.error || !result.userData) {
+              // Mark user inactive when fetch failed
+              try {
+                await userRepo.update({ address }, { isActive: false });
+                console.log(`Marked user ${address} as inactive (fetch error)`);
+              } catch (updateErr) {
+                console.error(`Failed to mark user ${address} inactive:`, updateErr);
```

ESLint

[error] Replace ``Failed·to·mark·user·${address}·inactive:`,·updateErr` with `⏎··················`Failed·to·mark·user·${address}·inactive:`,⏎··················updateErr⏎················`

(prettier/prettier)